adjutant_moc/actions/mailing_list.py

```diff
@@ -20,6 +20,51 @@
 from adjutant_moc.actions import base
 
 
+class Mailman(object):
+
+    def __init__(self, hostname, port, username, key):
+        self.hostname = hostname
+        self.port = port
+        self.username = username
+        self.key = key
+        self.client = paramiko.client.SSHClient()
+
+
+    def __enter__(self):
+        self.client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        self.client.connect(
+            hostname=self.hostname,
+            port=self.port,
+            username=self.username,
+            pkey=paramiko.RSAKey.from_private_key_file(self.key))
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.client.close()
+
+    def _execute(self, command):
+        stdin, stdout, stderr = self.client.exec_command(command)
+
+        errors = stderr.read()
+        if errors:
+            raise ConnectionError(errors)
+
+        # Note(knikolla): Not entirely sure if closing before reading is fine.
```

Comment:
is this comment old, or is it referring to something that exec_command is doing?

Reply:
That comment is old. I also realized that I didn't delete the old unused methods in the action, which I was keeping as a reference while splitting out the functionality to its separate class. Will delete both the comment and those now.